fulfillment-sdk/bin/settings.gradle

```diff
@@ -0,0 +1 @@
+rootProject.name = "inventory-sdk"
```

Comment:
fulfillment

Reply:
same thing...don't know what bin is being used for and why its not generated by the open api generator